### ISSUE TITLE
Minimap and integrated terminal rendered incorrectly on Arch Linux

### ISSUE THREAD

**Issue by deweiwork** [microsoft/vscode#74660]
- VSCode Version: 1.34.0
- OS Version: 5.1.5-arch1-2-ARCH ArchLinux

Steps to Reproduce:

![](https://i.imgur.com/24RcoLi.png)

Bug as shown in the picture above.

I think this is a problem with Electron,

because my other application has same problem too (haroopad),

this application is also based on electron.

My electron version is 4.1.4 .

Does this issue occur when all extensions are disabled?: Yes/No
yes

**Comment by JAicewizard** [comment]
dub of  #74378?

**Comment by deweiwork** [comment]
@JAicewizard  Yes, thanks!

**Comment by Tyriar** [comment]
Most of these issues I've been closing but I'll keep this one open for a bit so it's more discoverable.

The problem is related to your graphics drivers and/or Chromium not correctly blacklisting your configuration. Your current options are:

- Update your graphics drivers 🤞 
- Launch vscode using `code --disable-gpu`
- Disable canvas usage in terminal and minimap with these settings:
   ```
   "terminal.integrated.rendererType": "dom",
   "editor.minimap.enabled": false
   ```

This sort of issue should hopefully resolve itself when we update Electrn (https://github.com/microsoft/vscode/issues/61787). You can vote on https://github.com/microsoft/vscode/issues/15211 to make it more convenient to disable GPU usage.

**Comment by deweiwork** [comment]
Use code.desktop file to start vscode without gpu:
https://stackoverflow.com/questions/29966747/how-can-i-disable-gpu-rendering-in-visual-studio-code

**Comment by ranolfi** [comment]
Not actually relevant, but just adding - In my original report, I said I hadn't observed display output problems in any other application. Well, now I have. On Steam, which also uses Chromium (probably CEF). The Friends&Chat window won't open, and the Steam News window shows up blank.

![image](https://user-images.githubusercontent.com/8507649/59140955-308c6200-897b-11e9-9397-5e8739d33984.png)

**Comment by JAicewizard** [comment]
I have literally had that issue for like 2 years, its nothing new for me.
I have seen steam displaying the news/store as a bug at this point :P

I updated mesa to 19.0.6-1 and it fixed it for me(at least, for now)

**Comment by deweiwork** [comment]
@JAicewizard  @Tyriar 
Hi: 
I've also updated vscode and mesa to newest version with ArchLinux.
Minamap and integrated terminal are working normally now!

**Comment by ranolfi** [comment]
Confirmed fixed on [Mesa 19.0.6](https://www.mesa3d.org/relnotes/19.0.6.html). See [Bug 110761 - Huge problems between Mesa and Electron engine apps](https://bugs.freedesktop.org/show_bug.cgi?id=110761).
